EBA/Taschenrechner/Inforechner/Zahlensystem.cs

```diff
@@ -0,0 +1,191 @@
+
+            //von Dezimal in Ternär konvertieren
+            basis = 3;
+            var umgewandelt3 = "";
+            var ergebnis = Convert.ToInt32(umgewandelt10);
+            while (ergebnis != 0)
+            {
+                umgewandelt3 = Convert.ToString(ergebnis % basis) + umgewandelt3;
+                ergebnis = (ergebnis / basis);
+            }
+            /*MessageBox.Show(Convert.ToString(umgewandelt10));
+            MessageBox.Show(umgewandelt8);
+            MessageBox.Show(umgewandelt2);
+            MessageBox.Show(umgewandelt3);*/
+            return "Ausgangszahl=" + eingabe + ", im Dezimalsystem: " + umgewandelt10 + "im Binärsystem: " + umgewandelt2 + "im Ternärsystem: " + umgewandelt3 + "im Octalsystem: " + umgewandelt8;
+        }
+
+        public static string Dezimal(string eingabewert)
+        {
+            var eingabe = Convert.ToInt32(eingabewert);
+            var umgewandelt10 = eingabe;
+
+            //von Dezimal in Binär konvertieren
+            var basis = 2;
+            string umgewandelt2 = Convert.ToString(eingabe, basis);
+
+            //von Dezimal in Octal konvertieren
+            basis = 8;
+            string umgewandelt8 = Convert.ToString(eingabe, basis);
+
+            //von Dezimal in Ternär konvertieren
+            basis = 3;
+            var umgewandelt3 = "";
+            while (eingabe != 0)
+            {
+                umgewandelt3 = Convert.ToString(eingabe % basis) + umgewandelt3;
+                eingabe = (eingabe / basis);
+            }
+            /*MessageBox.Show(Convert.ToString(umgewandelt10));
+            MessageBox.Show(umgewandelt8);
+            MessageBox.Show(umgewandelt2);
+            MessageBox.Show(umgewandelt3);*/
+            return "Ausgangszahl=" + eingabe + ", im Dezimalsystem: " + umgewandelt10 + "im Binärsystem: " + umgewandelt2 + "im Ternärsystem: " + umgewandelt3 + "im Octalsystem: " + umgewandelt8;
+        }
+       
+        public static string Octal(string eingabewert)
+        {
+            //Konvertierung Octal in Dezimal
+            var umgewandelt10 = Convert.ToInt32(eingabewert, 8);
+
+            //Konvertierung Dezimal in Dual
+            var basis = 2;
+            string umgewandelt2 = Convert.ToString(umgewandelt10, basis);
+
+            //Konvertierung Dezimal in Ternär
+            basis = 3;
+            var eingabe = Convert.ToInt32(umgewandelt10);
+            var umgewandelt3 = "";
+            while (eingabe != 0)
+            {
+                umgewandelt3 = Convert.ToString(eingabe % basis) + umgewandelt3;
+                eingabe = (eingabe / basis);
+            }
+
+            /*MessageBox.Show(Convert.ToString(umgewandelt10));
+            MessageBox.Show(umgewandelt2);
+            MessageBox.Show(umgewandelt3);*/
+            return "Ausgangszahl=" + eingabe + ", im Dezimalsystem: " + umgewandelt10 + "im Binärsystem: " + umgewandelt2 + "im Ternärsystem: " + umgewandelt3;
```

Comment:
write a general output function

Reply:
this whole class got rm again ... @JanMichels  learn 2 merge